concordium-contracts-common/src/lib.rs

```diff
@@ -89,7 +89,11 @@ mod traits;
 mod impls;
 pub mod constants;
 pub mod schema;
+#[cfg(feature = "derive-serde")]
+pub mod schema_json;
 mod types;
+#[cfg(feature = "derive-serde")]
+pub mod versioned_schema_helpers;
```

Comment:
This should not be a public module. The only type that is exported from there should be moved to `schema` or `schema_json`.  This is an internal module only and pollutes the public API.